lab_1_keywords_tfidf/main.py

```diff
@@ -2,22 +2,35 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
-from typing import Optional, Union
+from typing import Optional, Union, List
+import re
 
 
-def clean_and_tokenize(text: str) -> Optional[list[str]]:
+
+def clean_and_tokenize(text: str) -> Optional[List[str]]:
     """
-    Removes punctuation, casts to lowercase, splits into tokens
+      Removes punctuation, casts to lowercase, splits into tokens
+      Parameters:
+      text (str): Original text
+      Returns:
+      list[str]: A sequence of lowercase tokens with no punctuation
+      In case of corrupt input arguments, None is returned
+      """
+    pass
```

Comment:
pass is only needed when there is no realisation, now that you have added it you can remove this line

Reply:
Done.